docs/ops/arithmetic/Divide_1.md

```diff
@@ -6,46 +6,46 @@
 
 **Short description**: *Divide* performs element-wise division operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Divide* does the following with the input tensors *a* and *b*:
+
+\f[
+o_{i} = a_{i} / b_{i}
+\f]
+
 **Attributes**:
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
-  * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+  * **Range of values**: refer to rule #1 in broadcast rules,description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
```

Comment:
please keep attributes explanation, for numpy refer to 'Broadcast Rules For Elementwise Operations'.

Reply:
fix